src/components/logo/Logo.jsx

```diff
@@ -0,0 +1,8 @@
+import logo from "./3.png";
+
+const Logo = (props) => {
+  const { classX } = props;
```

Comment:
change this wroking props name. I don't know what does it mean. Is it special class cause it's "X"?